docker-compose-insecure.yaml

```diff
@@ -0,0 +1,88 @@
+version: "3.8"
+services:
+  tinkerbell:
+    build:
+      context: ./cmd/tink-server/
+      dockerfile: Dockerfile
+    restart: unless-stopped
+    environment:
+      FACILITY: ${FACILITY:-onprem}
+      PACKET_ENV: ${PACKET_ENV:-testing}
+      PACKET_VERSION: ${PACKET_VERSION:-ignored}
+      ROLLBAR_TOKEN: ${ROLLBAR_TOKEN:-ignored}
+      ROLLBAR_DISABLE: ${ROLLBAR_DISABLE:-1}
+      PGDATABASE: tinkerbell
+      PGHOST: db
+      PGPASSWORD: tinkerbell
+      PGPORT: 5432
+      PGSSLMODE: disable
+      PGUSER: tinkerbell
+      TINKERBELL_GRPC_AUTHORITY: :42113
+      TINKERBELL_HTTP_AUTHORITY: :42114
+      TINKERBELL_INSECURE: "true"
```

Comment:
Do we need a full new docker-compoose? Can this not default to false and accept an override for true?

Reply:
Agree, I'll give this a shot